crates/bevy_image/src/image.rs

```diff
@@ -1128,21 +1127,31 @@ impl Image {
     /// Takes a 2D image containing vertically stacked images of the same size, and reinterprets
     /// it as a 2D array texture, where each of the stacked images becomes one layer of the
     /// array. This is primarily for use with the `texture2DArray` shader uniform type.
-    ///
-    /// # Panics
-    /// Panics if the texture is not 2D, has more than one layers or is not evenly dividable into
-    /// the `layers`.
```

Comment:
These should be changed into docs on what errors it might return.

Reply:
I did not see this with other functions. Here it could return every enum member. Should I still something to the comment here?

Reply:
We should be documenting errors and what cases might cause them, otherwise it may be difficult for consumers of these APIs to know how to handle them.

Reply:
Well this lead me down a rabbit hole. I just kept the original comment and changed to `# Errors`. Apparently this is sometimes a thing in rust and even has a clippy lint.
Not sure if this is used anywhere in bevy at the moment, but the documentation concerning errors seem inconsistent anyways. Let me know if this is good now.